src/js_lexer.zig

```diff
@@ -624,7 +624,7 @@ fn NewLexer_(
 
                         switch (lexer.code_point) {
                             // 0 cannot be in this list because it may be a legacy octal literal
-                            'v', 'f', 't', 'r', 'n', '`', '\'', '"', 0x2028, 0x2029 => {
```

Comment:
What if the string is `\\b`?  Or `\\u{2028}`? One where it does need to be escaped

Reply:
I might be misunderstanding you, but `\\b` should be interpreted as "literal backlash, b".  This sequence is actually in the test case.  Similar logic for the Unicode sequence, it should be interpreted as a literal backslash followed by u, {, 2, etc.